gost_ameth.c

```diff
@@ -57,10 +57,15 @@ static int pkey_bits_gost(const EVP_PKEY *pk)
 
     switch (EVP_PKEY_base_id(pk)) {
     case NID_id_GostR3410_2001:
+        return 256;
+#if defined(NID_id_GostR3411_2012_256)
     case NID_id_GostR3410_2012_256:
         return 256;
+#endif
+#if defined(NID_id_GostR3411_2012_256)
```

Comment:
They were defined at the same time in one commit. So if your patch is applied nothing but clarity will change.